### PR TITLE
Add definition for Legal Entity Basic Information

### PR DESCRIPTION
**Pull request by joakimnordling**
- Rremoved defintion related to ioxio-nexus-guides

### CODE REVIEW

**Review by lietu — CHANGES_REQUESTED** [review]
I think we should improve on this still

File: src/draft/LegalEntity/BasicInformation.py [hunk 1]
```diff
@@ -0,0 +1,4625 @@
...
+    ZW = "ZW"
+
+
+class LegalStatus(str, Enum):
+    NEW = "NEW"
+    ACTIVE = "ACTIVE"
+    DISBANDED = "DISBANDED"
+
+
+class RegisteredAddress(CamelCaseModel):
```
[the remark this answers is not in the document]
Comment (hsuomi) [reply src/draft/LegalEntity/BasicInformation.py:4552]: I agree we could have those character limits no initially. That is a good proposal and we can continue improving those limits further based on the results and feedback in the experimentation.

[the remark this answers is not in the document]
Comment (hsuomi) [reply src/draft/LegalEntity/BasicInformation.py:4552]: I was the most pondering between the buildingNumber / buildingIdentifier. It seemed that for a registered address separate from visiting and postal addresses the buildingNumber would have been enough in the Scandinavian context but I agree we should change the buildingNumber property still.

File: DataProducts/draft/LegalEntity/BasicInformation.json [hunk 2]
```diff
@@ -0,0 +1,4789 @@
...
+        "properties": {
+          "streetName": {
+            "title": "Street name",
+            "type": "string",
+            "example": "Fredrikinkatu"
+          },
+          "buildingNumber": {
+            "title": "Building number",
+            "type": "integer",
+            "example": 34
```
Comment (hsuomi) [inline DataProducts/draft/LegalEntity/BasicInformation.json:4736]: 54 A

**Review by hsuomi — CHANGES_REQUESTED** [review]

File: src/draft/LegalEntity/BasicInformation.py [hunk 3]
```diff
@@ -0,0 +1,4625 @@
...
+class BasicInformationRequest(CamelCaseModel):
+    national_identifier: str = Field(
+        ...,
+        title="National Identifier",
+        description="National identifier for a legal entity",
+        example="2464491-9",
+    )
+
+
+class LegalForm(str, Enum):
```
[the remark this answers is not in the document]
Comment (joakimnordling) [reply src/draft/LegalEntity/BasicInformation.py]: This is unfortunately not possible without changes to the tooling that is used to convert the definitions. I've suggested to create separate tasks for adding this support. Those changes should however be made all the way in the definitions template repo, not in this repo.

File: DataProducts/draft/LegalEntity/BasicInformation.json [hunk 4]
```diff
@@ -0,0 +1,4789 @@
...
+          "country"
+        ],
+        "type": "object",
+        "properties": {
+          "streetName": {
+            "title": "Street name",
+            "type": "string",
+            "example": "Fredrikinkatu"
+          },
+          "buildingNumber": {
```
[the remark this answers is not in the document]
Comment (joakimnordling) [reply DataProducts/draft/LegalEntity/BasicInformation.json]: Changed.

**Review by hsuomi — APPROVED** [review]

**Pull request merged**